app/grants/models.py

```diff
@@ -179,25 +234,43 @@ class Contribution(SuperModel):
 
     tx_id = models.CharField(max_length=255, default='0x0', help_text=_('The transaction ID of the Contribution.'))
     from_address = models.CharField(
-        max_length=255, default='0x0', help_text=_('The wallet address tokens are sent from.'),
+        max_length=255,
+        default='0x0',
+        help_text=_('The wallet address tokens are sent from.'),
     )
     to_address = models.CharField(
-        max_length=255, default='0x0', help_text=_('The wallet address tokens are sent to.'),
+        max_length=255,
+        default='0x0',
+        help_text=_('The wallet address tokens are sent to.'),
     )
     token_address = models.CharField(
-        max_length=255, default='0x0', help_text=_('The token address to be used with the Subscription.'),
+        max_length=255,
+        default='0x0',
+        help_text=_('The token address to be used with the Subscription.'),
     )
     token_amount = models.DecimalField(
-        default=1, decimal_places=4, max_digits=50, help_text=_('The promised contribution amount per period.'),
+        default=1,
+        decimal_places=4,
+        max_digits=50,
+        help_text=_('The promised contribution amount per period.'),
     )
     period_seconds = models.DecimalField(
-        default=0, decimal_places=0, max_digits=50, help_text=_('The number of seconds thats constitues a period.'),
+        default=0,
```

Comment:
This formatting looks like it's from `yapf`, but you're removing `yapf` from the pre-commit hook.  Do we want to leave this enabled?

Reply:
@mark Unfortunately, no, currently either isort or yapf need to be disabled, or we need to sort through their configurations and discern where they are conflicting and changing each other back and forth. It was preventing anything from being committed. On Friday I had removed isort and committed. I believe this commit is where the yapf formatting came from. After discussing the conflicting configurations with you briefly it sounded like isort might be the preferred linter, so I added isort back and removed yapf. Let me know if yapf is actually preferred.